app/src/main/java/com/globant/weatherapp/mvp/model/CityModel.kt

```diff
@@ -27,6 +33,7 @@ class CityModel(private val cities: String) : CityContracts.Model {
     }
 
     companion object {
+        const val FILE_NAME = "city.list.json"
```

Comment:
pls, can you rename the json file as city_list ?

Reply:
Yes, done!